src/current/v24.2/troubleshoot-lock-contention.md

```diff
@@ -0,0 +1,378 @@
+
+<img src="{{ 'images/v24.2/troubleshoot-lock-contention-waiting-statement.png' | relative_url }}" alt="Waiting statement" style="border:1px solid #eee;max-width:100%" />
+
+### Blocking transaction
+
+To identify the transaction that blocked **Transaction 2** and caused it to experience high contention, navigate back to the **Overview** tab.
+
+<img src="{{ 'images/v24.2/troubleshoot-lock-contention-overview-tab.png' | relative_url }}" alt="Overview tab" style="border:1px solid #eee;max-width:100%" />
+
+Scroll to the bottom of the Overview tab to the **Transaction with ID 452da3e7-e7fa-4801-aa40-c59988d14eb6 waited on** section which gives information about the blocking transaction.
+
+<img src="{{ 'images/v24.2/troubleshoot-lock-contention-blocking-transaction.png' | relative_url }}" alt="Blocking transaction" style="border:1px solid #eee;max-width:100%" />
+
+For more information about the blocking transaction, click the **Transaction Fingerprint ID** 08d8b7d97f830463 to open the [**Transaction Details** page]({% link {{ page.version.version }}/ui-transactions-page.md %}#transaction-details-page).
+
+<img src="{{ 'images/v24.2/troubleshoot-lock-contention-blocking-transaction-details.png' | relative_url }}" alt="Blocking transaction details" style="border:1px solid #eee;max-width:100%" />
+
+### Additional practice
+
+For Transaction 3, take steps similar to the above steps for Transaction 2, to identify the waiting statement that experienced high contention and the corresponding blocking transaction.
+
+## Remediate lock contention
+
+### Background context
+
+Locking conflicts are a natural artifact when business requirements call for concurrent data changes. Realistically, locking conflicts are unavoidable. The locking conflicts, however, are resolved efficiently with regard to the underlying resource utilization. When blocked transactions are waiting on a lock, they are not consuming CPU, disk, or network resources.
+
+Remediation is required when locking conflicts are too numerous, resulting in a significant increase in response time and/or decrease in throughput. Remediation of locking conflicts is typically about giving up some functionality in exchange for a reduction in locking contention. Example 2 uses two ways of doing this: [historical queries]({% link {{ page.version.version }}/as-of-system-time.md %}) and a ["fail fast" method]({% link {{ page.version.version }}/select-for-update.md %}#wait-policies). Use these remediations if they fit your application design. Other possible ways to reduce lock conflicts are: [column families]({% link {{ page.version.version }}/column-families.md %}) and [secondary indexes]({% link {{ page.version.version }}/schema-design-indexes.md %}).
+
+### Historical queries
```

Comment:
Perhaps it would be useful to have a small code block that contains the syntax for using those methods. Similar to how you have it in the Example 2 table below, just a bit earlier.  Just to have something to draw the eye towards the proposed remediation methods that are explained in more detail later, and make it super clear what the two options are.

e.g.:

### Historical queries

```
BEGIN AS OF SYSTEM TIME '-30s';
SELECT * FROM t2 WHERE k=4;
COMMIT;
```

Consider the following when using historical queries
...

### "fail fast" method

```
BEGIN;
SELECT * FROM t2 WHERE k=4 FOR UPDATE NOWAIT;
COMMIT;
```

Consider the following when using a "fail fast" method:
...